app/controllers/sessions_controller.rb

```diff
@@ -6,13 +6,47 @@ def create
     @user = find_user
 
     if @user&.mfa_enabled?
+      if @user.webauthn_credentials.any?
+        @webauthn_options = @user.webauthn_options_for_get
+
+        session[:webauthn_authentication] = {
+          challenge: @webauthn_options.challenge,
+          user: @user.display_id
```

Comment:
I'm not sure why `display_id` is used everywhere, seems like everyone has an `handle` and that `find_by_slug!` would find the first user without an handle if an user didn't have one

Reply:
everyone does not have handle. we have accounts that were imported from rubyforge, and some of them only have id.

Reply:
did a separate PR for that https://github.com/rubygems/rubygems.org/pull/2953